modules/nf-core/foldcomp/compress/tests/main.nf.test

```diff
@@ -0,0 +1,105 @@
+nextflow_process {
+
+    name "Test Process FOLDCOMP_COMPRESS"
+    script "../main.nf"
+    process "FOLDCOMP_COMPRESS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "foldcomp"
+    tag "foldcomp/compress"
+
+    test("proteomics - pdb") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['proteomics']['pdb']['tim1_pdb'], checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.fcz[0][1].find { file(it).name == "testA_0.fcz" })).match('testA_0')},
```

Comment:
Why snapshot only one file, don't know how the algorithm works but if the compression is deterministic maybe snapshot all the files?

Reply:
For some reason, they encode something in the later chains (other than the first) which sometimes change, so the md5sum breaks. However, the decompressed files' md5sum value afterwards is always the same (I will include these checks in a supplementary foldcomp decompress module).